### PR TITLE
Keep the margin between tasks after copy/past

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: app/scripts/proactive/view/dom.js [hunk 1]
```diff
@@ -1026,11 +1028,16 @@ define(
...
-                    copiedTasks = [];
+                   copiedTasks = [];
+                   positions = []
                     console.log("copy");
                     $(".selected-task").each(function (i, t) {
+                    positions.push({left: $(t).position().left, top: $(t).position().top})
                         copiedTasks.push($(t).data( "view" ))
                     })
+                    // let the user how he can do past(ctr-v)
+                     StudioClient.alert('alert-success', 'Click on where you want to do past and then do ctrl-V.', 'warning');
```
Comment (fviale) [inline app/scripts/proactive/view/dom.js]:
```suggestion
                     StudioClient.alert('Copy/paste', 'Click on the canvas where you want to paste and then do ctrl-V.', 'warning');
```

File: app/scripts/proactive/view/WorkflowView.js [hunk 2]
```diff
@@ -518,6 +517,19 @@ define(
...
+            var minLeft = 0;
+            var minTop = 0;
+            function sort(arr, type){
+                return Array.prototype.slice.call(arr).sort(function(a,b){
+                    return a[type] - b[type];
+                })
+            }
+            console.log(tasksPosition)
+            minLeft = sort(tasksPosition, 'left')[0].left;
+            minTop = sort(tasksPosition, 'top')[0].top;
```
Comment (fviale) [inline app/scripts/proactive/view/WorkflowView.js]: are all these logs necessary for debugging? Can you keep the essential part of it ?

Comment (medou-boushab) [reply app/scripts/proactive/view/WorkflowView.js]: done

Comment (medou-boushab) [reply app/scripts/proactive/view/WorkflowView.js]: No, I'll delete them!

**Comment by amouhoub** [comment]
@medou-boushab please do a Squash&Merge and not a Merge when merging this PR

**Pull request closed without merging**